tasks/build_tags.py

```diff
@@ -175,33 +190,35 @@
         "system-probe-unit-tests": SYSTEM_PROBE_TAGS.union(UNIT_TEST_TAGS),
```

Comment:
```suggestion
        "system-probe-unit-tests": SYSTEM_PROBE_TAGS.union(UNIT_TEST_TAGS).difference(UNIT_TEST_EXCLUDE_TAGS),
```
but maybe it's the only place where you don't want to exclude it?